src/main/java/com/github/bpiatek/bbghbackend/controller/ArticlesController.java

```diff
@@ -34,8 +37,9 @@
   @ApiResponses(value = {
       @ApiResponse(code = ORDINAL_200_OK, message = "Successfully retrieved all articles"),
   })
+  @ApiPageable
   @GetMapping
-  Page<Article> getAllArticlesPageable(Pageable pageable) {
+  Page<Article> getAllArticlesPageable(@ApiIgnore Pageable pageable) {
```

Comment:
jak dodales to `@ApiPageable` tutaj, to moze tez przy metodzie ponizej `Page<Comment> getAllCommentsForArticlePageable` warto to dodac ;)

Reply:
Tak.